src/transformation/visitors/function.ts

```diff
@@ -52,7 +53,12 @@ function isRestParameterReferenced(context: TransformationContext, identifier: l
     return references.some(r => !r.parent || !ts.isSpreadElement(r.parent) || !isVarargType(context, r));
 }
 
-export function transformFunctionBodyStatements(context: TransformationContext, body: ts.Block): lua.Statement[] {
+export function transformFunctionBodyStatements(context: TransformationContext, body: ts.ConciseBody): lua.Statement[] {
```

Comment:
`Statements` doesn't make much sense given that now it works on expressions too. `transformFunctionBodyContent`?

Reply:
thanks again @ark120202!